cpp/tests/utilities/column_utilities.hpp

```diff
@@ -40,8 +40,9 @@ void expect_column_properties_equal(cudf::column_view lhs, cudf::column_view rhs
  *
  * @param lhs The first column
  * @param rhs The second column
- */
-void expect_columns_equal(cudf::column_view lhs, cudf::column_view rhs);
+ * @param all If true display all differences
+ *---------------------------------------------------------------------------**/
+void expect_columns_equal(cudf::column_view lhs, cudf::column_view rhs, bool all = false);
```

Comment:
How about a clearer parameter name.

```suggestion
void expect_columns_equal(cudf::column_view lhs, cudf::column_view rhs, bool print_all_differences = false);
```

Reply:
OK.